api/envoy/service/status/v3/csds.proto

```diff
@@ -73,6 +89,7 @@ message PerXdsConfig {
       "envoy.service.status.v2.PerXdsConfig";
 
   ConfigStatus status = 1;
```

Comment:
It's probably worth adding a comment here that `status` will be populated by xDS servers but `client_side_config_status` will be populated by xDS clients.

Reply:
Comments added.

Reply:
(It's too bad that we can't use a `oneof` here, but that would be a breaking change for the go proto API.)

Reply:
Annotation added, TIL.

Reply:
@htuch Can you advice what should we use in v4alpha? The format suggests that the migration annotation is expected to be expanded into real `oneof` in v4alpha. But that will break backward compatibility.

Reply:
That's fine -- v4alpha will be a new major version (if it ever actually happens), so breaking changes are okay.  This is actually the whole point of the annotation, to make sure this change happens when the next major version bump occurs.

Reply:
Oh, but it looks like this didn't actually work for some reason.  The annotation was carried forward to v4alpha instead of being turned into a oneof.  Maybe the annotation has the wrong syntax or something?